packages/eslint-plugin-next/src/index.ts

```diff
@@ -105,8 +110,7 @@ const flatConfig = {
       ...coreWebVitalsRules,
     },
   },
-}
+} as const satisfies Record<string, Linter.Config>
 
-export default plugin
-export const { rules, configs } = plugin
-export { flatConfig }
+export default { ...plugin, flatConfigs: flatConfig }
+export { configs, flatConfig, rules }
```

Comment:
👀 I don't think these should be named exports, as this causes a types mismatch with the fact the plugin is CJS-only so you cannot access these at runtime?

Reply:
These values (`configs`, `flatConfig`, and `rules`) were already named exports, though. The change from

```ts
export const { rules, configs } = plugin
export { flatConfig }
```

to

```
export { configs, flatConfig, rules }
```

amounts to a no-op.

Reply:
Yah, I follow that, but I think both are wrong? With this, TypeScript will tell you that you can import them as named members, but at runtime you'll get an error that the module is CJS, so you can only use the default export?